osh/worker/csmock_runner.py

```diff
@@ -130,16 +129,32 @@ def analyze(self, analyzers, srpm_path, profile=None, su_user=None, additional_a
         if output_path:
             cmd += ' -o ' + shlex.quote(output_path)
 
+        return cmd
+
+    def handle_additional_arguments(self, additional_arguments):
```

Comment:
This is a static method

Reply:
I'm not sure I understand what you mean?